# HG changeset patch
# Parent af54deb96c090f4324a4ad92c628fa4716f40373
.hgignore

diff -r af54deb96c09 .hgignore
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/.hgignore	Sat Dec 06 23:39:47 2014 -0500
@@ -0,0 +1,12 @@
+syntax: regexp
+
+^objs/debug/
+^objs/release/
+
+^objs/win32/
+^objs/x64/
+
+^builds/win32/vc2013/freetype\.sdf$
+^builds/win32/vc2013/freetype\.opensdf$
+^builds/win32/vc2013/freetype\.v12\.suo$
+
