# HG changeset patch
# Parent 724286615987b5a8950c408c8212119dd3407fb7
Added .hgignore

diff -r 724286615987 .hgignore
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/.hgignore	Sun Nov 09 12:21:28 2014 -0500
@@ -0,0 +1,20 @@
+syntax: glob
+
+# VS maintenance files
+*.sdf
+*.opensdf
+*.suo
+
+syntax: regexp
+
+# build directories
+^projects/VS12/Debug/
+^projects/VS12/Release/
+^projects/VS12/x64/
+
+# user settings
+^projects/VS12/libmaxminddb\.vcxproj\.user$
+
+^include/maxminddb_config\.h$
+
+^projects/VS12/example\.c$
